apps/foundation-demo/src/app/design/molecules/sidebar/sidebar-content/sidebar-content.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'daff-sidebar-content',
+  template: '<ng-content></ng-content>',
+  host: {
+    class: 'daff-sidebar-content',
```

Comment:
I think this should be 'daff-sidebar__content'?

Reply:
So, I changed to this format since this somewhat acts as its own component. Though, you make a point because this leads to inconsistency with `daff-sidebar__item` since they're both built the same.

Reply:
I think for `daff` components `daff-sidebar-content` makes more sense, since `daff-sidebar__content` is specifically BEM

Reply:
actually I changed my mind. I'd prefer just sticking with BEM syntax anyway